tests/scripts/run-xunit-perf.cmd

```diff
@@ -107,8 +107,12 @@ setlocal
   )
 
   rem Save off the results to the root directory for recovery later in Jenkins
-  call :run_cmd xcopy "Perf-%BENCHNAME%*.xml" "%CORECLR_REPO%\" || exit /b 1
-  call :run_cmd xcopy "Perf-%BENCHNAME%*.etl" "%CORECLR_REPO%\" || exit /b 1
+  IF EXIST "Perf-%BENCHNAME%.xml" (
```

Comment:
We talked about this and decided to switch to using the %ETW_COLLECTION% as the variable to use.  We need to continue using copy because xcopy needs to specify whether you are copying a file or a directory.  We could do this by piping echo F to the xcopy, but just using copy seemed cleaner to me.